plugins/catalog-backend-module-bitbucket-cloud/config.d.ts

```diff
@@ -53,6 +55,11 @@ export interface Config {
                */
               projectKey?: RegExp;
             };
+            /**
+             * (Optional) TaskScheduleDefinition for the discovery.
+             * @visibility backend
```

Comment:
@freben re your comment at the other PR. Should I remove these here, too? (as it is the default)

Reply:
@pjungermann do it! :)

Reply:
done